openlibrary/core/vendors.py

```diff
@@ -102,29 +102,25 @@ def get_products(self, asins, serialize=False, marketplace='www.amazon.com',
         by comma or as a list.
         """
         item_ids = asins if type(asins) is list else [asins]
+        _resources = (self.RESOURCES[resources] if resources
+                      else self.RESOURCES['import'])
+        request = GetItemsRequest(partner_tag=self.tag,
+                                  partner_type=PartnerType.ASSOCIATES,
+                                  marketplace=marketplace,
+                                  item_ids=item_ids,
+                                  resources=_resources,
+                                  **kwargs)
```

Comment:
```suggestion
        try:
            request = GetItemsRequest(partner_tag=self.tag,
                                      partner_type=PartnerType.ASSOCIATES,
                                      marketplace=marketplace,
                                      item_ids=item_ids,
                                     resources=_resources,
                                     **kwargs)
        except ??:
            ??   
```

Reply:
https://user-images.githubusercontent.com/978325/77669173-a9c46800-6f41-11ea-82ea-4a2ba53c6bd6.png

Reply:
from paapi5_python_sdk.rest import ApiException
[...]
except ApiException as e:
    log(e)
    print(e)
    wait and try again?